HelpSource/Classes/SynthDef.schelp

```diff
@@ -229,39 +229,61 @@ x.free;
 
 subsection:: Argument Rates
 code::
-// the following two defs are equivalent. The first uses a 't_' arg:
 (
-SynthDef(\trigTest, { |out, t_trig=0, freq=440| // t_trig creates a TrigControl
+SynthDef(\trigTest1, { |out, t_trig=0, freq=440| // t_trig creates a TrigControl
 	Out.ar(out, SinOsc.ar(freq+[0,1], 0, Decay2.kr(t_trig, 0.005, 1.0)));
-}, [0, 4]		// lag the freq by 4 seconds (the second arg), but not t_trig (won't work anyway)
-);
+}, [0, 0, 4]		// lag the freq by 4 seconds (the second arg), but not t_trig (won't work anyway)
+).add;
 )
 
+
+// Using the first version
+y = Synth(\trigTest1);
+y.set(\t_trig, 1); // you can do this multiple times
+y.set(\t_trig, 1, \freq, 220); // hear how the freq lags
+y.free;
+
+// Pbind and Pmono know about the t_ notation so you should use the parameter trig
```

Comment:
Note that it's different for Pbind with the \set event type.

Pbind with the default \note event type, and Pmono, use an automatically-generated function to look up argument values. This function translates \trig in the event to \t_trig in the synth.

With event type \set, you list the desired arguments explicitly and there is no translation. Possibly there should be, but currently there's not.

```
Pbind(
    \type, \set,
    \id, ...,
    \args, #[t_trig],
    \t_trig, 1,
    \dur, ...
)
```

Reply:
Thanks ! I did not know that, I will change my PR accordingly

Reply:
I'm afraid I have to be a little bit fussy, for the sake of accuracy:

> Pbind and Pmono with the default \note event type

Technically, this should be "Pbind with the default \note event type, and Pmono, ..." -- Pmono uses event types \monoNote, \monoSet and \off, so it isn't exactly right to imply that Pmono uses \note.

Reply:
I like accuracy, especially in docs !
Thanks a lot for your review, I changed the file accordingly.